source/server-discovery-and-monitoring/tests/unified/closeInUse-pool-clear.yml

```
@@ -0,0 +1,322 @@
+---
+description: closeInUse
+
+schemaVersion: "1.11"
+
+runOnRequirements:
+  # failCommand appName requirements
+  - minServerVersion: "4.9"
+    serverless: forbid
+    topologies: [ replicaset, sharded, load-balanced ]
+
+createEntities:
+  - client:
+      id: &setupClient setupClient
+      useMultipleMongoses: false
+
+initialData: &initialData
+  - collectionName: &collectionName closeInUse
+    databaseName: &databaseName sdam-tests
+    documents: []
+
+tests:
+  - description: Connection pool clear uses closeInUseConnections=true after monitor timeout
+    operations:
+      - name: createEntities
+        object: testRunner
+        arguments:
+          entities:
+            - client:
+                id: &client client
+                useMultipleMongoses: false
+                observeEvents:
+                  - poolClearedEvent
+                  - connectionClosedEvent
+                  - commandStartedEvent
+                  - commandSucceededEvent
+                  - commandFailedEvent
+                  - connectionCheckedOutEvent
+                  - connectionCheckedInEvent
+                uriOptions:
+                  connectTimeoutMS: 500
+                  heartbeatFrequencyMS: 500
+                  appname: closeInUse
+                  retryReads: false
+                  minPoolSize: 0
+            - database:
+                id: &database database
+                client: *client
+                databaseName: *databaseName
+            - collection:
+                id: &collection collection
+                database: *database
+                collectionName: *collectionName
+            - thread:
+                id: &thread1 thread1
+      - name: insertOne
+        object: *collection
+        arguments:
+          document:
+            _id: 1
```

This is a reasonable place to use inline documents if you wish. I'll just make the suggestion here but you can apply it elsewhere if you like it:

```suggestion
          document: { _id: 1 }
```

My thinking here is the `arguments` and `document` are both relevant for someone reading the test file, but we probably don't care too much about the insert payload. Since it very easily fits on one line without running long, it's a candidate for inlining.